atom.js

```diff
@@ -526,8 +526,11 @@
 
 			// Set value for a key, or if `keyOrMap` is an object then set all the
 			// keys' corresponding values.
-			set: function (keyOrMap, value, validateFirst) {
+			set: function (keyOrMap, valueOrSetConfig, setConfig) {
 				var localConfig = config;
+				var isMap = isObject(keyOrMap);
+				var validateFirst = ((isMap ? valueOrSetConfig : setConfig) || {}).validate;
```

Comment:
I wouldn't store this as variable.
This would be better: `var config = Object.assign({ validate: true }, (isMap ? valueOrSetConfig : setConfig) || {})`.

Reply:
Sure, I'd skip the last empty object assignment though.